workflows/README.md

```diff
@@ -1 +1,58 @@
+# Getting started with Docker
+
+If you want to run these protocols interactively you will need to install Docker on your system. You can find more detailed instructions [here](https://docs.docker.com/install/). Alternatively we've developed snakefiles for automation that do not require installation. You can find those instructions below. 
+
+#### First, update your machine
+```
+sudo apt-get -y update && \
+sudo apt-get -y install python-pip \
+    zlib1g-dev ncurses-dev python-dev
+```
+#### Now, install the [open science framework command-line client](http://osfclient.readthedocs.io/en/stable/). We'll use this tool download data.
+```
+pip install osfclient
+```
+#### Finally, install [docker](https://www.docker.com)
```

Comment:
I could use a little more clarity around the docker workflow versus the snakefile + singularity workflow.

Above, it sounds like the snakefile + singluarity workflow doesn't require you to install Docker, but here you instruct the user to install Docker. Do you mean that the snakefile + singularity workflow doesn't require you to download and run all the container images locally?

I think that is the case, and that Docker is only used by Singularity in the instructions below.